### PR TITLE
Add support for IPTV Manager

### LINKED ISSUE #168: Integrate with Kodi PVR
Would it be possible to have a folder directly linking to all the channel live streams, cf the VRT Nu addon? This would allow for using this folder in widgets where the program info gets automatically updated as well. By "program info" I mean the information showing which program is currently being broadcast from when to when on that particular channel.

I can add all the direct links to the live streams of the different channels to my (super) favourites which is okay for starting the streams but it freezes the program info to the moment of creating the favourite. It would be great if I could immediately see from my widget which programs are being broadcast on the different channels like I can for the VRT Nu live streams.

Thanks!

### PR DESCRIPTION
**Pull request by michaelarnauts**
This PR adds support for exporting the Live channels and EPG data to the Kodi PVR by using `service.iptv.manager`.

See https://github.com/add-ons/service.iptv.manager

Screenshots:
image
image

Fixes #168

### CODE REVIEW

File: resources/lib/modules/kodi_pvr.py [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+import os
+from datetime import timedelta
+from functools import wraps
+
+from resources.lib.modules import CHANNELS
+from resources.lib.vtmgo.vtmgo import VtmGo
+from resources.lib.vtmgo.vtmgoepg import VtmGoEpg
+
+
+def file_output(routing):
```
Comment (michaelarnauts) [inline resources/lib/modules/kodi_pvr.py:16]: Still need to remove this.

**Comment by dagwieers** [comment]
I already merged the integration in VRT Radio and RegioTV add-ons. I hope we can do it tomorrow also for VRT NU and VTM GO.

**Comment by dagwieers** [comment]
VRT NU merged as well.

**Comment by michaelarnauts** [comment]
Let's do this! :)

**Comment by lovecrafthowardphillips** [comment]
Testing it with VRT NU, VTM GO and Regio TV and it works great already, excellent job! Love the channel hopping. The channel guide is only available for Eén, Canvas and Ketnet, though, is this normal? The guide is activated for all TV channels in the channel manager (I've deactivated the radio channels) with the client selected as the guide source.  When streaming a channel, it does show me the name of the programme I'm watching but nothing more.

**Comment by dagwieers** [comment]
We only have EPG data for Eén, Canvas and Ketnet. Not for Ketnet Jr or Sporza (which are both not official channels). Also for the regional channels we do not have EPG data (yet). For VRT Radio there is EPG data that we could also reuse for the radio live streams as well, but has not been written yet. Volunteers are welcome ;-)

**Comment by michaelarnauts** [comment]
@lovecrafthowardphillips don't you also have guide data for the VTM GO channels?

**Comment by lovecrafthowardphillips** [comment]
No, no guide data for any of the VTM GO channels which surprised me because in your screenshots it does show that data :)

**Comment by michaelarnauts** [comment]
Indeed. It should return data. Can you check your Kodi log?

**Comment by lovecrafthowardphillips** [comment]
Not sure what I should be looking for exactly but I've uploaded my log file [here](https://paste.kodi.tv/atinoxizad).

**Comment by michaelarnauts** [comment]
Not sure why VTM Go didn't respond, but you have so many errors related to piracy Addons that I'm not going to decipher it.

**Comment by lovecrafthowardphillips** [comment]
That's fine of course, I was just trying to help.

**Pull request merged**